corehq/apps/linked_domain/templates/linked_domain/tabs/pull_release_content_tab.html

```diff
@@ -0,0 +1,80 @@
+{% load hq_shared_tags %}
+{% load i18n %}
+
+<div id="ko-tabs-pull-content" class="ko-template">
+  <div data-bind="if: master_link">
+    <p>
+      {% blocktrans %}
+        This project is linked to the upstream project <a data-bind="attr: {'href': master_href}, text: master_link.master_domain"></a>.<br/>
+        The following items below are synced from the upstream project.<br/>
+        <a href="#" target="_blank">Learn more</a> about Linked Projects.
+      {% endblocktrans %}
+    </p>
+
+    <div class="panel panel-default">
+      <div class="panel-heading">
+        <h3 class="panel-title">{% trans 'Linked Data Models' %}</h3>
+      </div>
+      <div class="panel-body">
+        <table class="table table-striped table-hover">
+          <thead>
+          <tr>
+            <th class="col-sm-5">{% trans "Linked Model" %}</th>
+            <th class="col-sm-2">{% trans "Last Updated" %} ({{ timezone }})</th>
+            <th class="col-sm-5"></th>
+          </tr>
+          </thead>
+          <tbody data-bind="foreach: model_status">
+          <tr>
+            <td data-bind="text: name"></td>
+            <td data-bind="text: lastUpdate"></td>
+            <td data-bind="css: {'has-error': error}">
+              <button class="btn btn-danger" data-toggle="modal" data-bind="visible: showUpdate() && !update_url, attr: {'data-target': '#overwrite-content-modal-' + $index()}">
+                <i class="fa fa-refresh"></i> {% trans "Overwrite" %}
```

Comment:
@biyeun I ended up sticking with just the "Overwrite" text because it feels like the "sync" action is implied.

Reply:
I wouldn't think so? I think Overwrite feels more destructive/scary, but `Sync & Overwrite` feels more inviting? This feels like a potential UAT question.

Reply:
Yeah that is fair. Changed it to `Sync & Overwrite` for now.